charts/eks/templates/syncer.yaml

```diff
@@ -106,8 +106,12 @@ spec:
       {{- if .Values.volumes }}
 {{ toYaml .Values.volumes | indent 8 }}
       {{- end }}
+      {{- if .Values.syncer.storage.binariesVolume }}
+{{ toYaml .Values.syncer.storage.binariesVolume | indent 8 }}
+      {{- else }}
         - name: binaries
           emptyDir: {}
+      {{- end }}
```

Comment:
I'd instead just set the default in the `.Values.syncer.storage.binariesVolume` and add a `toYaml` here instead of the if/else here, as it adds more lines and more complexity to the templating.

Reply:
Yeah, sounds reasonable. I change it now.